src/xtopcom/xcontract_common/src/xcontract_execution_context.cpp

```diff
@@ -909,7 +909,7 @@ uint64_t xtop_contract_execution_context::calc_decayed_tgas() const {
     uint64_t decayed_tgas{0};
     auto last_hour = contract_state()->last_tx_hour();
     auto height = contract_state()->time();
-    auto decay_time = XGET_ONCHAIN_GOVERNANCE_PARAMETER(usedgas_decay_cycle);
+    auto decay_time = XGET_ONCHAIN_GOVERNANCE_PARAMETER(usedgas_reset_interval);  //usedgas_decay_cycle Deprecated
```

Comment:
the Deprecated name of usedgas_decay_cycle  should not to usedgas_decay_cycle_deprecated, which can indicate that the configration name is deprecated